app/models/manageiq/providers/vmware/infra_manager/metrics_capture.rb

```diff
@@ -265,22 +273,30 @@ def perf_release_vim
     @perf_vim_hist = @perf_vim = nil
   end
 
+  def with_vim_perf_history
+    raise "No block given" unless block_given?
+
```

Comment:
did you forget a yield?

Reply:
:man_facepalming: I started on another PR without changing branches